webapp/public/style_domjudge.css

```diff
@@ -203,6 +203,13 @@ img.affiliation-logo {
 }
 .scoreheader th {
     text-align: center;
+    box-shadow: -1px 0px 0px 0px silver inset,
+                0px 2px 0px 0px black;
+    border: none;
+    background: white;
```

Comment:
Does the background have to be white for this to work?

Reply:
![image](https://user-images.githubusercontent.com/14887731/80916572-ced6a280-8d59-11ea-9a52-00f36dde32b0.png)
I need a background, but there is this: https://caniuse.com/#feat=css-variables, so the background is now equal to the bg of the body but explicit